grazel-gradle-plugin/src/main/kotlin/com/grab/grazel/migrate/kotlin/KotlinProjectDataExtractor.kt

```diff
@@ -66,19 +66,13 @@ constructor(
         val srcs = project.kotlinSources(sourceSets, SourceSetType.JAVA_KOTLIN).toList()
         val resources = project.kotlinSources(sourceSets, SourceSetType.RESOURCES).toList()
 
-        val deps = projectDependencyGraphs
-            .directDependencies(
-                project,
-                BuildGraphType(ConfigurationScope.BUILD)
-            ).map { dependent ->
-                gradleDependencyToBazelDependency.map(project, dependent, null)
-            } +
-            dependenciesDataSource.collectMavenDeps(
-                project,
-                BuildGraphType(ConfigurationScope.BUILD)
-            ) +
-            project.androidJarDeps() +
-            project.kotlinParcelizeDeps()
+        val deps = projectDependencyGraphs.directDependencies(
```

Comment:
Is this formatting correct? Default Kotlin formatting puts each params in separate line no?